test/env/util.go

```diff
@@ -53,14 +53,14 @@ users:
 `
 )
 
-// RootDirOrSkip returns the ROOTDIR environment variable, or skips if the
+// VariableOrFail returns the specified environment variable, or fails if the
 // variable is undefined or empty.
-func RootDirOrSkip(t *testing.T) string {
-	rootDir := os.Getenv("ROOTDIR")
-	if len(rootDir) == 0 {
-		t.Skip("WARNING: skipping test as 'ROOTDIR' is not defined")
+func VariableOrFail(t *testing.T, name string) string {
```

Comment:
```suggestion
func GetenvOrFail(t *testing.T, name string) string {
```

I got a bit confused by this function name. The docs (and code) were clarifying, but I think the func name could be improved.